ui/src/app/stores/workflows/workflows.reducers.ts

```diff
@@ -269,5 +269,29 @@ export function workflowsReducer(state: State = initialState, action: WorkflowsA
       };
     default:
       return state;
+    case WorkflowsActions.RUN_WORKFLOW:
+      return {
+        ...state,
+        workflowAction: {
```

Comment:
I did this mistake, It doesn't make sense have 
  workflowAction: {
          ...state.workflowAction,
        },
there :D 
overrides old state with old state

Reply:
Ok, admittedly I just copy-pasted that. I thought it's needed because the `...` doesn't do a deep copy.